config/locales/en/mailers.yml

```diff
@@ -27,6 +27,18 @@ en:
         COPY of booking rejection for %{prisoner}
       copy_of_rejection: >-
         This is a copy of the booking rejection email sent to the visitor
+    cancelled:
+      subject: "%{status}: %{prisoner} on %{date}"
+      salutation: Dear staff,
+      details: >-
+        %{visitor} has cancelled the visit to %{prisoner} (%{prisoner_number) on
+        %{date}
+      reference: "Reference number: %{reference_no}"
+      please_delete: Please delete this visit from NOMIS.
+      closing: |
```

Comment:
Ah, yeah, it was a semi-text message in the old app and I was considering just making it `text.erb`.